### DISCUSSION TITLE
Whitelisting IP's

### DISCUSSION THREAD

**Discussion by visma-kimhjortholm** [upptime/upptime#169] [Q&A] [answered]
Will the requests from upptime come from a fixed number of IP's thus making it possible to include those IP adresses in a whitelist

**Comment by AnandChowdhary** [comment, marked as answer]
Yes, they will always come from GitHub Actions' IP addresses, which are also Azure IP addresses, available here: https://docs.github.com/en/free-pro-team@latest/actions/reference/specifications-for-github-hosted-runners#ip-addresses.
